e2e/frozen_deps_test.go

```diff
@@ -37,8 +37,8 @@ var _ = Describe("E2E TEST:FrozenDeps", func() {
 				Logf:        l.Logf,
 			}
 			result := checks.FrozenDeps(&checkRequest)
-			Expect(result.Error).Should(BeNil())
-			Expect(result.Pass).Should(BeTrue())
+			Expect(result.Error).ShouldNot(BeNil())
```

Comment:
we should also add a case where it passes. tensorflow can fix this some day, so we usually try for passing repos?